tests_disabled.json

```diff
@@ -205,6 +205,10 @@
             "name" : "selftests.test_deproxy.DeproxyTestH2.test_bodyless_multiplexed",
             "reason" : "Need to investigate"
         },
+	{
+            "name" : "selftests.test_deproxy.DeproxyTestH2.test_bodyless_multiplexed_multiple",
+            "reason" : "Disabled by issue #1377#issuecomment-1210615335"
```

Comment:
This is bit confusing: selftests are to test the framework itself, but disabling a test means that it actually tests Tempesta FW. I think this test should be moved out from selftests